### DISCUSSION TITLE
Not an Issue - a question

### DISCUSSION THREAD

**Discussion by ghostinthecamera** [rebtd7/FF13Fix#61] [Q&A]
Hi there - apologies this is not an issue but there isn't really another proper way to contact you about something.

I'm currently developing a set of photographic tools for FFXIII-2 having done so for FFXIII. These are essentially a free camera, a timestop (to pause the game so you can take shots) and removal of the HUD, DOF and Bloom.

I have all of this for FFXIII and am looking to recreate for FFXIII-2. This is a long shot but i wanted to ask if you had discovered any part of the games code which handles the timescale of the game? I.e. a value which if set to 0 would pause the game, or NOPing an instruction (this was the method for FFXIII however the same instruction is difficult to find in FFXIII-2).

Thanks for your time!
ghost

**Comment by rebtd7** [comment]
Unfortunately I did not find anything of that sort. I can take a look when I have more time
